project/TestExtras.scala

```diff
@@ -54,11 +54,18 @@ object TestExtras {
           def shouldExist(description: String, filename: String): Unit =
             require(file(filename).exists, s"$description should be run as part of the build")
 
-          List(
-            "The java JavaExtension.java" -> "akka-actor-tests/target/test-reports/TEST-akka.actor.JavaExtension.xml",
+          val baseList =
+            List(
+              "The java JavaExtension.java" -> "akka-actor-tests/target/test-reports/TEST-akka.actor.JavaExtension.xml")
+          val jdk9Only = List(
             "The jdk9-only FlowPublisherSinkSpec.scala" -> "akka-stream-tests/target/test-reports/TEST-akka.stream.scaladsl.FlowPublisherSinkSpec.xml",
             "The jdk9-only JavaFlowSupportCompileTest.java" -> "akka-stream-tests/target/test-reports/TEST-akka.stream.javadsl.JavaFlowSupportCompileTest.xml")
-            .foreach((shouldExist _).tupled)
+
+          val testsToCheck =
+            if (isJdk8) baseList
+            else baseList ::: jdk9Only
```

Comment:
I thought it would make more sense to check it here then fiddle with yml configs.